compiler-rt/lib/scudo/standalone/tests/secondary_test.cpp

```diff
@@ -336,6 +336,31 @@ TEST_F(MapAllocatorCacheTest, CacheOrder) {
     MemMap.unmap();
 }
 
+TEST_F(MapAllocatorCacheTest, CacheOrderWithRelease) {
+  std::vector<scudo::MemMapT> MemMaps;
+  Cache->setOption(scudo::Option::MaxCacheEntriesCount,
+                   CacheConfig::getEntriesArraySize());
+
+  fillCacheWithSameSizeBlocks(MemMaps, CacheConfig::getEntriesArraySize() - 1,
+                              TestAllocSize);
+
+  // Enable releases and perform a deallocation to trigger releases
```

Comment:
Isn't the release triggered by the call to store?

Reply:
Yes, when `fillCacheWithSameSizeBlocks()` is called, this will trigger a deallocation which calls `store()`. Did you want me to put the comment about this directly above the call to `fillCacheWithSameSizeBlocks()`?

Reply:
Update: we will now directly trigger releases with `releaseToOS()`.